src/libs/actions/Report.js

```diff
@@ -689,39 +689,67 @@ function addComment(reportID, text) {
 
 /**
  * Gets the latest page of report actions and updates the last read message
+ * If a chat with the passed reportID is not found, we will create a chat based on the passed participantList
  *
  * @param {String} reportID
+ * @param {Array} participantList The list of users that are included in a new chat, not including the user creating it
+ * @param {Object} newReportObject The optimistic report object created when making a new chat, saved as optimistic data
  */
-function openReport(reportID) {
+function openReport(reportID, participantList = [], newReportObject = {}) {
+    const onyxData = {
+        optimisticData: [{
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {
+                isLoadingReportActions: true,
+                lastVisitedTimestamp: Date.now(),
```

Comment:
I know that this wasn't introduced in this PR and I won't block on it, but using `Date.now()` generates a timestamp with the current user's timezone. Just noting that this might cause bugs with unread messages.

Reply:
Is there a standard practice for getting the correct UTC timestamp?

Reply:
Nvm, `Date.now()` seems to return the unix timestamp which is already UTC.